{{cookiecutter.project_slug}}/modules/utils.py

```diff
@@ -6,6 +6,12 @@
 GLOBAL_OPTIONS_FILE_PATH = f"{Path.cwd()}/modules/options.json"
 
 
+def posixpath_to_modulepath(posixpath):
+    parent_mondule_path = posixpath.parent.as_posix().split("/")
+    module_name = [posixpath.stem, ]
+    return ".".join(parent_mondule_path + module_name) 
```

Comment:
```suggestion
    return f"{parent_module_path}.{module_name}"
```
Perhaps an f-string here is more readable?

Reply:
parent_module_path is a list, to return with a f-string it would have to be:
```
parent_module_path = ".".join(posixpath.parent.as_posix().split("/"))
module_name = posixpath.stem
return f"{parent_module_path}.{module_name}"
```
perhaps it's a bit more readable, something like:
```
def posixpath_to_modulepath(posixpath):
    module_parent_path = ".".join(posixpath.parent.as_posix().split("/"))
    return f"{module_parent_path}.{posixpath.stem}"
```
?

Reply:
Last one looks very easy to read and understand. If that is working I would suggest that one